Frameworks/Foundation/NSURLRequest.mm

```diff
@@ -138,6 +140,8 @@ - (void)encodeWithCoder:(NSCoder*)coder {
     [coder encodeBool:_HTTPShouldUsePipelining forKey:@"HTTPShouldUsePipelining"];
     [coder encodeBool:_HTTPShouldHandleCookies forKey:@"HTTPShouldHandleCookies"];
     [coder encodeBool:_allowsCellularAccess forKey:@"allowsCellularAccess"];
+
+    [coder encodeObject:_properties forKey:@"_properties"];
```

Comment:
It is fine to encode nil; if there is nothing in properties there's no point in storing an empty dictionary